test.js

```diff
@@ -129,7 +139,7 @@ it('normal throw', function() {
     'throw new Error("test");'
   ], [
     'Error: test',
-    /^    at Object\.exports\.test \(.*\/line1\.js:1001:101\)$/
+    /^    at Object\.exports\.test \((?:.*\/)?line1\.js:1001:101\)$/
```

Comment:
I see, this is unfortunate because we should test whether our paths are relative or absolute, clients may be relying on this feature…

Maybe `compareStackTrace()` should be updated to be able to specify 2 regexps for a given line if the native and translated traces are different.

@LinusU Any opinion on this?